packages/react/src/index.js

```diff
@@ -66,6 +66,7 @@ export FileUploader, {
 export { FilterableMultiSelect } from './components/FilterableMultiSelect';
 export Form from './components/Form';
 export FluidForm from './components/FluidForm';
+export FluidTextInput from './components/FluidTextInput';
```

Comment:
I think we want these to be experimental from the start

```suggestion
export FluidTextInput as unstable__FluidTextInput from './components/FluidTextInput';
```